css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
+	background-size: contain;
+	margin-left: 50px;
+	margin-top: 20px;
+	float: left;
+	margin-right: 150px;
+}
+
+header{
+	width: 100%;
+	height: 80px;
+	box-shadow: 0px 3px 10px grey;
```

Comment:
I don't think there would be any difference. I guess it could be also just a plain zero. I left the unit because on Chrome developer tools I can play with Up/Down arrows, if I want to test things. Maybe I'm wrong.